app/controllers/nonsipp/memberdetails/SchemeMembersListController.scala

```diff
@@ -17,7 +17,7 @@
 package controllers.nonsipp.memberdetails
 
 import services.PsrSubmissionService
-import pages.nonsipp.memberdetails.SchemeMembersListPage
```

Comment:
Are any changes needed in SchemeMembersListControllerSpec to check the completed pages filtering?

Reply:
Added a test for this